src/testFixtures/workspaceFolder/go1-plain-sam-app/template.yaml

```diff
@@ -0,0 +1,43 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+    go1-plain-sam-app
+
+    Sample SAM Template for go1-plain-sam-app
+
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
+Globals:
+    Function:
+        Timeout: 5
+
+Resources:
+    HelloWorldFunction:
+        Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+        Properties:
+            CodeUri: hello-world/
+            Handler: hello-world
+            Runtime: go1.x
+            Tracing: Active # https://docs.aws.amazon.com/lambda/latest/dg/lambda-x-ray.html
+            Events:
+                CatchAll:
+                    Type: Api # More info about API Event Source: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#api
+                    Properties:
+                        Path: /hello
+                        Method: GET
+            Environment: # More info about Env Vars: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#environment-object
+                Variables:
+                    PARAM1: VALUE
+
+Outputs:
```

Comment:
For our test fixtures we can strip out environment variables, output, and test (at least that is how we do it in JB: https://github.com/aws/aws-toolkit-jetbrains/blob/master/testdata/samProjects/image/go1.x/template.yaml#L4

Reply:
The current debug config tests check for these fields so they'll be left for now.